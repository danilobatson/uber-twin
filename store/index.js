import { configureStore } from "@reduxjs/toolkit";
import { combineReducers } from "redux";
import navReducer from "./slices/navSlice.js";
import userReducer from "./slices/userSlice";

const rootReducer = combineReducers({
    nav: navReducer,
    user: userReducer,
});

const store = configureStore({
    reducer: rootReducer,
});

export default store;

