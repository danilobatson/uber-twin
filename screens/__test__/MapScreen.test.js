import React from 'react';
import renderer from 'react-test-renderer';

import { render, screen } from '@testing-library/react-native';

import MapScreen from '../MapScreen';
import { Provider } from 'react-redux';
import store from '../../store';

describe('<MapScreen />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <MapScreen />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});