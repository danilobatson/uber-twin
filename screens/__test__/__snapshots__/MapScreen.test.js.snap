// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapScreen /> renders correctly 1`] = `
<View>
  <Text>
    Here is the map stuff...
  </Text>
  <View />
  <View
    style={
      {
        "height": "50%",
      }
    }
  />
  <View
    style={
      {
        "height": "50%",
      }
    }
  />
</View>
`;
